plugins/Preprocessor/fileDefinitions/preprocessorScript.json

```diff
@@ -0,0 +1,11 @@
+{
+	"id": "preprocessorScript",
+	"type": "text",
+	"icon": "mdi-language-javascript",
+	"detect": {
+		"packType": "behaviorPack",
+		"scope": "preprocessor_scripts/",
+		"matcher": ["preprocessor_scripts/**/*", "preprocessor_scripts/**/*"],

```

Comment:
This should probably be `["preprocessor_scripts/**/*", "preprocessor_scripts/*"]`; I've just noticed that we have this issue within some of our JS files as well

Reply:
Yes, you're right